File: 1018/1018.java
// bee 1018 - Cedulas
// Alonso Martins
// 27/09/2023
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int[] valoresNotas = {100, 50, 20, 10, 5, 2, 1};
        int[] notasUsadas = new int[7];
        int valor = in.nextInt();
        System.out.println(valor);
        for(int i = 0; i < valoresNotas.length; i ++) {
            notasUsadas[i] = valor/valoresNotas[i];
            valor %= valoresNotas[i];
        }
        for(int i = 0; i < notasUsadas.length; i ++) {
            System.out.printf("%d nota(s) de R$ %d,00\n", notasUsadas[i], valoresNotas[i]);
        }
    }
}

File: 1018/1018.c
// bee 1018 - Cedulas
// Alonso Martins
// 03/07/2022
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);
    int notas[7] = {
        100, 50, 20, 10, 5, 2, 1
    };
    char cedulas[7][10] = {
        "100,00",
        "50,00",
        "20,00",
        "10,00",
        "5,00",
        "2,00",
        "1,00"
    };
    printf("%d\n", n);
    for (int i = 0; i < 7; i ++) {
        printf("%d nota(s) de R$ %s\n", (n/notas[i]), cedulas[i]);
        n %= notas[i];
    }
    return 0;
}
